apps/arweave/src/ar_entropy_storage.erl

```diff
@@ -356,6 +367,19 @@ record_entropy(ChunkEntropy, BucketEndOffset, StoreID, RewardAddr) ->
 	end,
 
 	release_semaphore(Filepath).
+
+%% @doc If we are not at the beginning of the entropy, shift the offset to
+%% the left. store_entropy will traverse the entire 2.9 partition shifting
+%% the offset by sector size. It may happen some sub-chunks will be written
```

Comment:
> It may happen some sub-chunks will be written to the neighbouring storage module(s) on the left or on the right since the storage module may be configured to be smaller than the partition.

This is no longer relevant, right?

Reply:
yep! Sorry I have a edit locally that removes that comment, was just waiting on e2e tests to pass